css/properties/border-image-repeat.json

```diff
@@ -0,0 +1,114 @@
+{
+  "css": {
+    "properties": {
+      "border-image-repeat": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/border-image-repeat",
+          "support": {
+            "chrome": {
+              "version_added": "15"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "15"
+            },
+            "firefox_android": {
+              "version_added": "15"
+            },
+            "ie": {
+              "version_added": "11"
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "safari": {
+              "version_added": "6"
+            },
+            "safari_ios": {
+              "version_added": "9.3",
+              "notes": "See <a href='https://bugs.webkit.org/show_bug.cgi?id=14185'>WebKit bug 14185</a>."
```

Comment:
This note, from the original table, is citing the bug prompting implementation. It's more of a historical point than informative about compatibility per se. Not sure if it's worth keeping.

Reply:
Not worth keeping. Please remove.